src/helpers/daily-data-providers/apple-health-sleep.ts

```diff
@@ -110,10 +117,26 @@ function coreSleep(startDate: Date, endDate: Date, value: "Asleep" | "InBed") {
 	});
 }
 
-export function asleepTime(startDate: Date, endDate: Date) {
+export function asleepOnlyAsleepTime(startDate: Date, endDate: Date) {
 	return coreSleep(startDate, endDate, "Asleep");
 }
 
 export function inBedTime(startDate: Date, endDate: Date) {
 	return coreSleep(startDate, endDate, "InBed");
+}
+
+export function asleepCoreTime(startDate: Date, endDate: Date) {
+	return coreSleep(startDate, endDate, "AsleepCore");
+}
+
+export function asleepRemTime(startDate: Date, endDate: Date) {
+	return coreSleep(startDate, endDate, "AsleepREM");
+}
+
+export function asleepDeepTime(startDate: Date, endDate: Date) {
```

Comment:
We should register these 3 new apple health sleep types in the daily data types:
https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/helpers/query-daily-data.ts#L119

https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/helpers/query-daily-data.ts#L181

(I would not do "asleep only"; don't think that is useful - just add core, rem, and deep)